scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -63,7 +67,7 @@ ENV HOME /home/sof
 RUN mkdir -p /home/sof/work/alsa && cd /home/sof/work/alsa && \
 git clone $CLONE_DEFAULTS https://github.com/thesofproject/alsa-lib.git && \
 git clone $CLONE_DEFAULTS https://github.com/thesofproject/alsa-utils.git && \
-cd /home/sof/work/alsa/alsa-lib && ./gitcompile &&  make install && \
+cd /home/sof/work/alsa/alsa-lib && ./gitcompile --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu/ &&  make install && \
```

Comment:
Is this related to #2543? Some other bug? Please add a comment or URL.

Reply:
no we don't have any issue link. This issue was found after upgrading to Ubuntu 20.04. I referred to build from the scratch (https://thesofproject.github.io/latest/getting_started/build-guide/build-from-scratch.html) Do you want me to add this to commit text?

Reply:
The docs are not good either. Maybe just copy the error message if you don't this into a comment?

Reply:
already commit text has oneline error message.

```
[  9%] Generating sof-cnl-rt274.tplg
alsatplg: error while loading shared libraries: libatopology.so.2: cannot open shared object file: No such file or directory
make[2]: *** [topology/CMakeFiles/topology_sof-cnl-rt274.dir/build.make:61: topology/sof-cnl-rt274.tplg] Error 127
```

by default, alsa libraries go to /usr/lib64.

Reply:
I think this error deserves more visibility and to be mentioned not just in the commit message but in a comment too. It's just a comment so you don't have to re-test :-)

Testing changes here is very time-consuming, one reason why these configuration files need even more comments than usual IMHO.

Reply:
yes, I agree. I added that as comments also.